Bug/namespace the css

leakymemory opened the pull request:
We want to make sure that the CSS provided with the component doesn't conflict with any of the other page elements.

msftclas commented:
Hi __@mttwc__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

leakymemory committed:
Namespace the component CSS

We want to make sure that the CSS provided with the component doesn't conflict
with any of the other page elements.

leakymemory committed:
Fix the indent.

anbinm-msft commented on sampleapp/index.html:
```diff
@@ -7,7 +7,7 @@
 	</head>
 	<body style="min-height:100%">
-		<div id="oneNotePicker" style="width:295px;height:180px;margin:50px;overflow-y:scroll"></div>
+		<div id="oneNotePicker" style="width:350px;height:350px;margin:50px;overflow-y:scroll"></div>
```
Ideally the inline style should be extracted out

anbinm-msft reviewed (approved):
LGTM
